lld/COFF/Driver.h

```diff
@@ -86,8 +86,8 @@ class LinkerDriver {
 
   void addClangLibSearchPaths(const std::string &argv0);
 
-  // Used by the resolver to parse .drectve section contents.
-  void parseDirectives(InputFile *file);
+  // Used by the resolver to iterate through .drectve section(s).
+  void processDrectveSections(InputFile *file);
```

Comment:
Same comment throughout: we could just call this `processDirectivesSection`. Same above and below.